html/Corporate/index.html

```diff
@@ -265,19 +272,40 @@ <h4 class="feature__subtitle">Some awesome features</h4>
     </header>
 
     <!-- Form -->
-    <form class="contact__form contact-form">
-      <input class="contact-form__input" type="text" placeholder="FULL NAME">
+    <form class="contact__form contact-form" name="contact__form" method="POST">
 
-      <input class="contact-form__input" type="email" placeholder="YOUR EMAIL">
+      <!-- Name -->
+      <div class="contact-form__input-box">
+        <input class="contact-form__input" name="username" type="text" placeholder="FULL NAME">
+        <div id="contact-form__name-error" class="contact-form__val-error"></div>
+      </div>
+      
+      <!-- Email -->
+      <div class="contact-form__input-box">
+        <input class="contact-form__input" name="email" type="email" placeholder="YOUR EMAIL">
+        <div id="contact-form__email-error" class="contact-form__val-error"></div>
```

Comment:
don't use IDs as part of BEM because it might be confusing for other developers and can create a mess in css selectors. For IDs you can use nouns like "info-block" (or "infoBlock" depends on your project agreements)